pkg/proxy/apis/config/types.go

```diff
@@ -22,6 +22,17 @@ import (
 	logsapi "k8s.io/component-base/logs/api/v1"
 )
 
+// IPVSIPSetConfiguration contains ipset-related configuration
+// details for IPVS proxier.
+type IPVSIPSetConfiguration struct {
+	// hashSize is the size of hash table for ipset
+	// +optional
+	HashSize *int32
+	// maxElem is the maximal number of elements that can be stored in an ipset
+	// +optional
+	MaxElem *int32
```

Comment:
Maybe `MaxElements` instead?
(which can internally translate to maxelem in utilipset)